.github/workflows/create-compare-reference.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           ref: ${{ env.GIT_REFERENCE }}
           fetch-depth: 0
 
-      - uses: prefix-dev/setup-pixi@v0.4.3
+      - uses: prefix-dev/setup-pixi@v0.5.0
         with:
           cache-write: ${{ github.event_name == 'push' && github.ref_name == 'master' }}
```

Comment:
you need `environments: test` here. Otherwise it will only cache the `default` environment.